### PR TITLE
Update README - for VSCode Marketplace

### PR DESCRIPTION
**Pull request by vincentjocodes**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,76 +1,80 @@
...
-[11]: https://cloud.google.com/code/docs/vscode/troubleshooting
-[12]: https://github.com/GoogleCloudPlatform/cloud-code-vscode/raw/master/images/app_deploy_debug.gif
-[13]: https://join.slack.com/t/googlecloud-community/shared_invite/zt-erdf4ity-8ZMUQ18DYV~5hkbZ~gCswg
-[14]: https://www.gstatic.com/cloudssh/cloudcode/client-library-browser.gif
-[15]: https://www.gstatic.com/cloudssh/cloudcode/secret-manager-explorer.png
+Cloud Code for VS Code brings the power and convenience of IDEs to cloud-native Kubernetes and Cloud Run application development. Cloud Code works with Google’s command-line [container tools](https://github.com/GoogleContainerTools) like [`skaffold`](https://skaffold.dev/) and [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl/) under the hood, providing local, continuous feedback on your project as you build, edit, and run your applications locally or in the cloud.
+
+![E2E workflow](https://github.com/GoogleCloudPlatform/cloud-code-vscode/raw/master/images/cloud-code-quick-deploy.gif)
+
+List of Supported Services
```
Comment (sanguinecloud) [inline README.md:7]: You can remove this list of supported services (it was meant to be part of a larger structural change)

File: README.md [hunk 2]
```diff
@@ -1,76 +1,80 @@
...
+
+## Supporting your development workflow
+
+Get run-ready samples, out-of-the-box configuration snippets, support for key Google services like Google Cloud APIs and Cloud Build, one click deployment, a tailored debugging experience, and much more with Cloud Code — making developing with Kubernetes and Cloud Run a whole lot easier!
+
+<details>
+  <summary>Read more</summary>
+  
+  ### Highlights:
+  - Pick your preferred language with Cloud Code’s support for Go, Java, Node.js, Python, .NET Core app development.
```
Comment (sanguinecloud) [inline README.md:27]: For the time being, add these points under here:

- Get straight to developing with Cloud Code’s simplified authentication workflow that uses your Google Cloud credentials.
- Monitor your app with streaming logs and customize the output with additional filters to produce results meaningful to you.

File: README.md [hunk 3]
```diff
@@ -1,76 +1,80 @@
...
+
+## Kubernetes development
+
+Create and run a new app in minutes with Cloud Code’s Kubernetes support. Or work on an existing application, customize its YAML with Cloud Code’s intelligent authoring support, debug it relentlessly with the setup-free debugger, and run it on any of your Kubernetes clusters. Whatever your workflow is, Cloud Code helps you spend less time on configuration and context-switching, so you can focus on developing your app.
+
+<details>
+  <summary>Read more</summary>
+
+  ### Highlights:
+  - Get started with in-built ready-to-run starter Kubernetes apps for your favorite languages and frameworks. [Read the docs](https://cloud.google.com/code/docs/vscode/creating-an-application)
```
Comment (sanguinecloud) [inline README.md:38]: I've been told in-built is a ye olde English word. 'built-in' here.

File: README.md [hunk 4]
```diff
@@ -1,76 +1,80 @@
...
+## Containerization made easy
+Create secure, production-ready container images from source code without having to worry about a Dockerfile with Cloud Code’s built-in support for Google Cloud Buildpacks. You get to focus on building your application, not containerizing it. 
+
+## Built-in Secret Manager support
+Protect sensitive information and keep your app secure with Cloud Code’s integrated Secret Manager support. You can create, view, update, and use secrets in the Secret Manager view without having them in your codebase. [Read the docs](https://cloud.google.com/code/docs/vscode/secret-manager)
+
+## Google Cloud APIs at your fingertip
+Browse available Cloud APIs, enable services, and install and learn how to integrate client libraries in your app without leaving your IDE and breaking your development flow, all with the API library browser. [Read the docs](https://cloud.google.com/code/docs/vscode/client-libraries)
+
+- Get straight to developing with Cloud Code’s simplified authentication workflow that uses your Google Cloud credentials.
```
Comment (sanguinecloud) [inline README.md:68]: You can drop lines 68-73, since it's been added to the first list up top.

File: README.md (hunk 1 above)
Comment (vincentjocodes) [reply README.md:7]: Thanks. Removed in the new PR.

File: README.md (hunk 2 above)
Comment (vincentjocodes) [reply README.md:27]: Done.

File: README.md (hunk 3 above)
Comment (vincentjocodes) [reply README.md:38]: Done

File: README.md (hunk 4 above)
Comment (vincentjocodes) [reply README.md:68]: Done. Thanks.

**Comment by vincentjocodes** [comment]
Thanks Sangeetha! Here is the updated based on your review: https://github.com/GoogleCloudPlatform/cloud-code-vscode/pull/349

**Pull request merged**